pkg/scheduler/scheduler.go

```diff
@@ -80,7 +80,7 @@ type Scheduler struct {
 	NextPod func() *framework.QueuedPodInfo
 
 	// Error is called if there is an error. It is passed the pod in
-	// question, and the error
+	// queue, and the error
```

Comment:
I think `in question` is more accurate, you need think that pod is not in the queue now.

Reply:
thanks. done